lms/djangoapps/ccx/api/v0/tests/test_views.py

```diff
@@ -422,7 +423,7 @@ def test_post_list_wrong_master_course_special_cases(self):
         self.mstore.update_item(self.course, self.coach.id)
 
         # case with deprecated  master_course_id
-        with mock.patch('lms.djangoapps.courseware.courses.get_course_by_id', autospec=True) as mocked:
+        with mock.patch('lms.djangoapps.ccx.api.v0.views.get_course_by_id', autospec=True) as mocked:
```

Comment:
this change I know nothing about, but assume it's right if tests are passing

Reply:
Yeah, tests were failing without this. It's indirect/opaque, but basically this works around import order issues.

Instead of patching the copy of the function directly in the `courses` module (where it's defined), we need to patch it (the imported copy) inside the module (`views`) that's actually using it here in the test. The function has presumably already been imported by the time we start executing code here.